xbmc/cores/VideoPlayer/VideoRenderers/OverlayRenderer.h

```diff
@@ -94,6 +94,19 @@ namespace OVERLAY {
     float m_height;
     float m_source_width{0}; // Video source width resolution used to calculate aspect ratio
     float m_source_height{0}; // Video source height resolution used to calculate aspect ratio
+
+  protected:
+    /*!
+     * \brief Given the resolution ratio determines if it is a 4/3 resolution
+     * \param resRatio The resolution ratio (the results of width / height)
+     * \return True if the ratio refer to a 4/3 resolution, otherwise false
+     */
+    bool IsSquareResolution(float resRatio)
+    {
+      // 1:25 is the ratio of old resolutions like PAL (e.g. 720x576)
+      // 1:33 is the ratio of the most common 4/3 resolutions
+      return resRatio == 1.25f || resRatio == 1.33f;
```

Comment:
Why 1.33f and not 1.333f or 1.3333f ?

4/3 != 1.33

Reply:
I did not obtain reports with infinite value
but i can add same proposed change

Reply:
change applied